diff --git a/node_modules/@radix-ui/react-tooltip/dist/index.d.ts b/node_modules/@radix-ui/react-tooltip/dist/index.d.ts
index 03af161..72b4135 100644
--- a/node_modules/@radix-ui/react-tooltip/dist/index.d.ts
+++ b/node_modules/@radix-ui/react-tooltip/dist/index.d.ts
@@ -3,17 +3,7 @@ import { DismissableLayer } from '@radix-ui/react-dismissable-layer';
 import * as PopperPrimitive from '@radix-ui/react-popper';
 import { Portal as Portal$1 } from '@radix-ui/react-portal';
 import { Primitive } from '@radix-ui/react-primitive';
-
-type Scope<C = any> = {
-    [scopeName: string]: React.Context<C>[];
-} | undefined;
-type ScopeHook = (scope: Scope) => {
-    [__scopeProp: string]: Scope;
-};
-interface CreateScope {
-    scopeName: string;
-    (): ScopeHook;
-}
+import type { CreateScope } from '@radix-ui/react-context';
 
 declare const createTooltipScope: CreateScope;
 interface TooltipProviderProps {
