diff --git a/node_modules/@radix-ui/react-dropdown-menu/dist/index.d.ts b/node_modules/@radix-ui/react-dropdown-menu/dist/index.d.ts
index eddc22e..38907a5 100644
--- a/node_modules/@radix-ui/react-dropdown-menu/dist/index.d.ts
+++ b/node_modules/@radix-ui/react-dropdown-menu/dist/index.d.ts
@@ -1,17 +1,7 @@
 import * as React from 'react';
 import { Primitive } from '@radix-ui/react-primitive';
 import * as MenuPrimitive from '@radix-ui/react-menu';
-
-type Scope<C = any> = {
-    [scopeName: string]: React.Context<C>[];
-} | undefined;
-type ScopeHook = (scope: Scope) => {
-    [__scopeProp: string]: Scope;
-};
-interface CreateScope {
-    scopeName: string;
-    (): ScopeHook;
-}
+import type { CreateScope } from '@radix-ui/react-context';
 
 type Direction = 'ltr' | 'rtl';
 declare const createDropdownMenuScope: CreateScope;
